hokusai/deployment.py

```diff
@@ -0,0 +1,70 @@
+import datetime
+import json
+
+from hokusai.config import config
+from hokusai.kubectl import Kubectl
+from hokusai.common import print_red, print_green, shout, get_ecr_login
+
+class Deployment(object):
```

Comment:
i would go for more of a command object pattern here, so instead of `update` the method would be `run` or something like that.